CesiumGeospatial/include/CesiumGeospatial/EarthGravitationalModel96Grid.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "Library.h"
+
+#include <CesiumGeospatial/Cartographic.h>
+
+#include <gsl/span>
+
+#include <cstdint>
+#include <optional>
+#include <string>
+#include <vector>
+
+namespace CesiumGeospatial {
+
+/**
+ * @brief Allows elevation data in the format provided by the Earth
+ * Gravitational Model 1996 to be loaded and queried.
+ */
+class CESIUMGEOSPATIAL_API EarthGravitationalModel96Grid final {
+public:
+  /**
+   * Attempts to create a {@link EarthGravitationalModel96Grid} from the given file.
+   *
+   * This method expects a file in the format of the WW15MGH.DAC 15-arcminute
+   * grid.
+   */
+  static std::optional<EarthGravitationalModel96Grid>
+  fromFile(const std::string& filename);
+
+  /**
+   * Attempts to create a {@link EarthGravitationalModel96Grid} from the given buffer.
+   *
+   * This method expects the buffer to contain the contents of the WW15MGH.DAC
+   * 15-arcminute grid.
+   */
+  static std::optional<EarthGravitationalModel96Grid>
+  fromBuffer(const gsl::span<const std::byte>& buffer);
+
+  /**
+   * Samples the height at the given position.
+   *
+   * @param position The position to sample.
+   * @returns The height representing the difference in meters of mean sea-level
+   * from the surface of a WGS84 ellipsoid.
```

Comment:
Can we state this a little more clearly? The current wording doesn't make it clear to me whether the returned value is positive or negative when the EGM96 surface is above the ellipsoid. Something like:

"The height (in meters) of the EGM96 surface above the WGS84 ellipsoid. A positive value indicates that the EGM96 surface is above the ellipsoid, while a negative value indicates that the MSL surface is below the ellipsoid."

(Please double check I have that right! I can never remember if it's that or the opposite. 😆)

Reply:
It's surprisingly hard to find out which one it is, but it seems like other tools loading EGM96 treat positive as above the ellipsoid.